youtube_dl/YoutubeDL.py

```diff
@@ -1879,6 +1879,8 @@ def ensure_dir_exists(path):
                         except (OSError, IOError):
                             self.report_error('Cannot write subtitles file ' + sub_filename)
                             return
+                    elif sub_info.get('downloader') is not None:
```

Comment:
Would `callable(sub_info.get('downloader'))` be a safer test?

Reply:
Good one. Done.